gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -1358,8 +1413,9 @@ def main(args):
                 header_dict = pickle.load(f)
 
             if test:
-                logger.info("Filtering to test partitions on chr20, X, and Y...")
-                ht = filter_to_test(ht)
+                if data_type != "joint":
```

Comment:
The test I have is from that gene on chr22, and it will be empty.

Reply:
Because we recently added 'test' in this one: https://github.com/broadinstitute/gnomad_qc/blob/9e6218e373a99b03924548fdfe166e30d108db38/gnomad_qc/v4/create_release/validate_and_export_vcf.py#L210

Reply:
Ah right I forgot we dropped the other test case in the release. This is fine for now as it generates test for the test gene but our original validation tests were for ch22,chrX, and chrY because that subset allowed us to test all of our validity checks. The export doesnt need those specific contigs so this way works. Lets just drop this test filtering for exomes and genomes too, so delete L1408-1410